Workbooks/CosmosDb Fleet/Overview.workbook

```diff
@@ -0,0 +1,2315 @@
+                    "formatter": 8,
+                    "formatOptions": {
+                      "min": 0,
+                      "palette": "blue",
+                      "aggregation": "Sum"
+                    },
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "maximumFractionDigits": 1
+                      },
+                      "emptyValCustomText": "0"
+                    }
+                  }
+                ],
+                "rowLimit": 10000,
+                "filter": true,
+                "hierarchySettings": {
+                  "treeType": 1,
+                  "groupBy": [
+                    "Subscription"
+                  ],
+                  "expandTopLevel": true,
+                  "finalBy": "Name"
+                },
+                "sortBy": [
+                  {
+                    "itemKey": "$gen_heatmap_microsoft.documentdb/databaseaccounts-Requests-TotalRequests_3",
+                    "sortOrder": 2
+                  }
+                ],
+                "labelSettings": [
+                  {
+                    "columnId": "Subscription",
+                    "label": "Subscription/DatabaseAccount"
+                  },
+                  {
+                    "columnId": "Name",
+                    "label": "Collections"
+                  },
+                  {
+                    "columnId": "microsoft.documentdb/databaseaccounts-Requests-TotalRequests",
+                    "label": "Requests"
+                  },
+                  {
+                    "columnId": "microsoft.documentdb/databaseaccounts-Requests-TotalRequests Timeline",
+                    "label": "Requests Timeline"
+                  }
+                ]
+              },
+              "sortBy": [
+                {
+                  "itemKey": "$gen_heatmap_microsoft.documentdb/databaseaccounts-Requests-TotalRequests_3",
+                  "sortOrder": 2
+                }
+              ],
+              "showExportToExcel": true
+            },
+            "conditionalVisibility": {
+              "parameterName": "selectedTab",
+              "comparison": "isEqualTo",
+              "value": "Operation"
+            },
+            "showPin": true,
+            "name": "cosmosDB account operations"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "### Start editing the Pool section here\r\n"
+            },
+            "conditionalVisibility": {
+              "parameterName": "1",
+              "comparison": "isEqualTo",
+              "value": "2"
+            },
+            "name": "OperationSectionEditStarts - Copy"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "Only refer to this tab if you have configured your fleetspace to consume pooled RU/s.",
```

Comment:
This does not have conditional visibility set on it? I assume it should only be visible when the pool tab is selected